### ISSUE TITLE
How do I limit the number of OTAA attempts with retry parameter?

### ISSUE THREAD

**Issue by heroichornet** [Lora-net/LoRaMac-node#359]
I want to limit the number of join attempts to 1. Its not working and I was wondering how to do this.

I tried modifying this parameter in the MAC-request:
https://github.com/Lora-net/LoRaMac-node/blob/f42be67be402a40b3586724800771bfe13fb18e6/src/mac/LoRaMac.c#L3153

Also I don't understand why it has to be bigger than 48 in order to be verified. 
https://github.com/Lora-net/LoRaMac-node/blob/f42be67be402a40b3586724800771bfe13fb18e6/src/mac/region/RegionEU868.c#L396

In short, please point me in the right direction so when I do a OTAA join request I get only one attempt by the MAC.

**Comment by heroichornet** [comment]
closed because duplicate
